project/Dependencies.scala

```diff
@@ -82,6 +85,8 @@ object Dependencies {
 
     val aeronDriver = "io.aeron" % "aeron-driver" % aeronVersion // ApacheV2
     val aeronClient = "io.aeron" % "aeron-client" % aeronVersion // ApacheV2
+    // Added explicitly for when artery tcp is used
+    val agrona = "org.agrona" % "agrona" % agronaVersion // ApacheV2
```

Comment:
good, maybe add additional note up at the version numbers that it should be updated if the aeronVersion is updated so we use same as the aeron dependency